website/src/pages/api/chat/index.ts

```diff
@@ -13,7 +13,17 @@ const handler = withoutRole("banned", async (req, res, token) => {
     if (req.query.chat_id) {
       data = await client.get_chat(req.query.chat_id as string);
     } else {
-      data = await client.get_my_chats();
+      const params = {};
+      if (req.query.limit) {
```

Comment:
Can you remove this option and hard code it to this file?

Reply:
Done